### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by 0x00A1FA** [victorgarciaesgi/nuxt-typed-router#67] [General]
Hi,

Just wanted to say a quick "thank you" for this great package. It's really useful and will surely be one of the main reasons why I'll be sticking with Nuxt.

God bless you!

P.S.: [1 John 5:20](https://www.kingjamesbibleonline.org/1-John-5-20/)

**Comment by victorgarciaesgi** [comment]
Thank you!
